service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -108,6 +113,155 @@ public void recordStewardship(
     record.store();
   }
 
+  @Override
+  public void addSite(JWTData userData, AddSiteRequest addSiteRequest) {
+    SitesRecord sitesRecord = db.newRecord(SITES);
+
+    sitesRecord.setBlockId(addSiteRequest.getBlockId());
+    sitesRecord.setLat(addSiteRequest.getLat());
+    sitesRecord.setLng(addSiteRequest.getLng());
+    sitesRecord.setCity(addSiteRequest.getCity());
+    sitesRecord.setZip(addSiteRequest.getZip());
+    sitesRecord.setAddress(addSiteRequest.getAddress());
+
+    sitesRecord.store();
+
+    SiteEntriesRecord siteEntriesRecord = db.newRecord(SITE_ENTRIES);
```

Comment:
oof this is a lot. it might be easier and cleaner to do something like `db.newRecord(SITE_ENTRIES, addSiteRequestDTO)` (also I think the way that method works is similar to Jackson, so make sure an empty constructor is available if it's throwing exceptions)

Reply:
The `addSiteRequest` contains more info than the table `SITE_ENTRIES` takes, so I would still have to individually select which fields to add.